### ISSUE TITLE
DataTable: This role must have an accessible name.

### ISSUE THREAD

**Issue by KubaGorka** [primefaces/primereact#8289]
### Describe the bug

The DataTable component fails accessibility requirements (E950). An element with role='table' must have an accessible name. Setting name via props doesn't set it correctly.

### System Information

```Shell
primereact: 10.9.7
```

### Steps to reproduce the behavior

1. Go to https://primereact.org/multiselect/
1. Edit any example via stackblitz by adding prop `name`
1. Notice that the it is not set correctly

### Expected behavior

Passing `name` prop should set it on the correct element
